data/json/construction.json

```
@@ -141,6 +141,28 @@
     "pre_terrain": "t_door_frame",
     "post_terrain": "t_door_c"
   },
+  {
+    "type": "construction",
+    "description": "Fill Shallow Water With Dirt",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "survival", 1 ] ],
+    "time": "60 m",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "components": [ [ [ "material_soil", 60 ] ] ],
```

```suggestion
    "components": [ [ [ "material_soil", 60 ], [ "material_sand", 400 ] ] ],
```

This would fill it with a roughly equivalent volume of sand as a second option.

---

But... why would the sand result in a dirt tile?

---

The only difference functionally between sand and dirt is biomatter, and pools of still water are full of that.